src/cozmo/annotate.py

```diff
@@ -146,6 +146,35 @@ def add_img_box_to_image(image, box, color, text=None):
             text.render(d, (x1, y1, x2, y2))
 
 
+def add_polygon_to_image(image, poly_points, scale, line_color, fill_color=None):
+    '''Draw a polygon on an image
+
+    This will draw a polygon on the passed in image in the specified
+    colors and scale.
+
+    Args:
+        image (:class:`PIL.Image.Image`): The image to draw on
+        poly_points: A sequence of points representing the the polygon,
+            where each point has float members (x, y)
+        scale (float): Scale to multiply each point by to match the image scaling
```

Comment:
Regardining "Scale to multiply each point by to match the image scaling"

Should we remove the 'by' so that it is: "Scale to multiply each point to match the image scaling"?